app/src/main/java/com/codesoom/assignment/TaskAdvice.java

```diff
@@ -0,0 +1,60 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.exceptions.TaskNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+
+import java.util.Date;
+
+/**
+ * 할 일들의 예외처리를 위한 클래스
+ *
+ * @see TaskNotFoundException
+ * @see MethodArgumentNotValidException
+ * @see ErrorResponse
+ */
+@RestControllerAdvice
+public class TaskAdvice {
+
+    /**
+     * TaskNotFoundException 이 발생하면 404코드와 에러정보를 리턴한다.
```

Comment:
주석이 코드를 너무 자세하게 설명하면 커플링이 생깁니다. 이 메소드의 내용이 수정되었을 때 주석도 바뀌지 않도록 하는 것이 좋습니다.
```suggestion
     * 할 일을 찾을 수 없는 경우의 예외를 처리합니다.
```